.ruby-version

```diff
@@ -1 +1 @@
-2.3.3
+2.4.0
```

Comment:
Lets leave 2.3.3 for now because elastic beasntalk doesn't work yet with 2.4.0 :/